php-practice_1.php

```diff
@@ -0,0 +1,126 @@
+<?php
+// Q1 変数と文字列
+$name = '奥西';
+echo  '私の名前は「'. $name .'」です。' . "\n";
```

Comment:
[nits]
 ```.```の前後の空白が抜けています！

細かいですが、コードが統一されていると可読性が良くなり
保守性が高まるので常に意識するようにしましょう！
```suggestion
echo  '私の名前は「' . $name . '」です。' . "\n";
```

Reply:
空白を設ける対応をしました。